src/sagemaker/jumpstart/curated_hub/hub_client.py

```diff
@@ -0,0 +1,85 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""This module contains a client with helpers to access the Private Hub."""
+from __future__ import absolute_import
+import boto3
+from botocore.exceptions import ClientError
+
+from sagemaker.jumpstart.types import JumpStartModelSpecs
+
+
+class CuratedHubClient:
+    """Calls SageMaker Hub APIs for the curated hub."""
+
+    def __init__(self, curated_hub_name: str, region: str) -> None:
+        """Sets up region and underlying client."""
+        self.curated_hub_name = curated_hub_name
+        self._region = region
+        self._sm_client = boto3.client("sagemaker", region_name=self._region)
+
+    def create_hub(self, hub_name: str, hub_s3_bucket_name: str):
+        """Creates a Private Hub."""
+        hub_bucket_s3_uri = f"s3://{hub_s3_bucket_name}"
+        self._sm_client.create_hub(
+            HubName=hub_name,
+            HubDescription="This is a curated hub.",  # TODO verify description
```

Comment:
can we store these as a constant somewhere

Reply:
+1